app/assets/stylesheets/apply.scss.erb

```diff
@@ -1,45 +1,86 @@
+.card-style {
+  background-color: white;
+  box-shadow: 0 0 5px 2px #ddd;
+  border: 1px solid #bbb;
+}
+
 .panel {
-  height: 100vh;
-  text-align:center;
+  top: 0;
   width:100%;
-  min-height:500px;
+  height:500px;
+  position: fixed;
+  z-index: -1;
+
+  .panel-container-left &, .panel-container-right & {
+    width: 50%;
+    height: 100vh;
+    @media only screen and (max-width: 45em) {
+      position: absolute;
+      top: initial;
+      width: 100%;
+    }
+  }
+
+  .panel-container-left & {
+    float: left;
+  }
+
+  .panel-container-right & {
+    float: right;
+  }
 }
 
-.nonprofits-bg {
-  background:
-  -webkit-linear-gradient(
-  rgba(255, 255, 255, 0.3),
-  rgba(255, 255, 255, 0.3)
-  ),
-  url(<%= image_path 'apply/nonprofits/nonprofits.jpg' %>);
-  background-size:cover;
+.panel-container{
+  @extend .panel;
+  margin-top: -48px;
+  text-align:center;
+  position: static;
+  z-index: 0;
+
+  .title {
+    @extend .h0;
+    color: #FFFFFF;
+    padding-top: 200px;
+  }
 }
 
-.students-bg {
-  background:
-  -webkit-linear-gradient(
-  rgba(27, 117, 188, 0.3),
-  rgba(27, 117, 188, 0.3)
-  ),
-  url(<%= image_path 'apply/students/students.jpg' %>);
-  background-size:cover;
+.panel-container-left, .panel-container-right {
+  @extend .panel-container;
+  width: 50%;
+
+  @media only screen and (max-width: 45em) {
+    width: 100%;
+  }
 }
 
-#left-panel, #right-panel {
-    height: 100vh;
-    text-align:center;
-    width: 50%;
-    @media only screen and (max-width: 45em) {
-        width: 100%;
-    }
+.panel-container-left {
+  float: left;
 }
 
-#left-panel {
-    float: left;
+.panel-container-right {
+  float: right;
 }
 
-#right-panel {
-    float: right;
+.nonprofits-bg {
+  @include background-image(
+  linear-gradient(
+  rgba(50, 50, 50, 0.4),
+  rgba(50, 50, 50, 0.4)
+  ),
+  url(<%= image_path 'apply/nonprofits/nonprofits.jpg' %>));
+  background-size:cover;
+  left: 0;
+}
+
+.students-bg {
+  @include background-image(
+  linear-gradient(
+  rgba(27, 117, 188, 0.4),
+  rgba(27, 117, 188, 0.4)
+  ),
+  url(<%= image_path 'apply/students/students.jpg' %>));
```

Comment:
You can just use `image-url("apply/students/students.jpg")` for this instead of using ERB (same for the one above)